src/Microsoft.SqlTools.ServiceLayer/ServerConfigurations/ServerConfigService.cs

```diff
@@ -239,5 +239,27 @@ private ConfigProperty GetSmoConfig(Server server, int configNumber)
                 throw new ServerConfigException($"Failed to get config. config number: ${configNumber}", ex);
             }
         }
+
+        public ServerConfigProperty GetConfigByName(Server server, string configName)
+        {
+            try
+            {
+                ConfigProperty serverConfig = null;
+                foreach (ConfigProperty configProperty in server.Configuration.Properties)
+                {
+                    if (configProperty.DisplayName == configName)
```

Comment:
Why are we using display name for this instead of some sort of non-localized value?

Reply:
Change it to use the property number instead of the displayName